git/lib/init.rb

```diff
@@ -1,10 +1,21 @@
 begin
-  require File.expand_path("../../../lib/environment_light", __FILE__)
-  require File.expand_path("../../../config/initializers/paths", __FILE__)
+  require File.expand_path('../../../lib/environment_light', __FILE__)
+
+  if AppConfig::getName != 'Settings'
```

Comment:
Style/ColonMethodCall: Do not use `::` for method calls.<br>Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.